takwimu/templates/homepage.html

```diff
@@ -102,6 +102,20 @@ <h2 class="display-4 font-weight-bold mb-4">{{ country|title }}</h2>
     .country-container:nth-of-type(even) {
       background-color: #f4f4f4;
     }
+    .breadcrumb {
+      padding: 1rem;
+    }
+    .breadcrumb-item {
+      padding-left: .1rem;
+      letter-spacing: 0;
+      font-size: 14px;
+    }
+    .breadcrumb-item+.breadcrumb-item::before {
+        display: inline-block;
+        padding-right: .1rem;
+        color: #fff;
+        content: ">>";
```

Comment:
I think this is an actual glyph and not just less than signs i.e »

Reply:
@DavidLemayian I have replaced this with glyph